docs/plus/config-map.md

```diff
@@ -13,12 +13,24 @@ and will therefore not be included in the resulting manifest.
 
 ```typescript
 import * as kplus from 'cdk8s-plus-25';
+import { Construct } from 'constructs';
+import { App, Chart, ChartProps } from 'cdk8s';
 
-const config: kplus.IConfigMap = kplus.ConfigMap.fromConfigMapName('config');
+export class MyChart extends Chart {
+  constructor(scope: Construct, id: string, props: ChartProps = { }) {
+    super(scope, id, props);
 
-// the 'config' constant can later be used by API's that require an IConfigMap.
-// for example when creating a volume.
-const volume = kplus.Volume.fromConfigMap(config);
+    const config: kplus.IConfigMap = kplus.ConfigMap.fromConfigMapName(this, 'config-map', 'config');
+
+    // the 'config' constant can later be used by API's that require an IConfigMap.
+    // for example when creating a volume.
+    kplus.Volume.fromConfigMap(this, 'volume', config);
+  }
+}
+
+const app = new App();
+new MyChart(app, 'config-map');
```

Comment:
```suggestion
new MyChart(app, 'VolumeFromConfigMap');
```

What do you think about using a more descriptive name like this?

Reply:
That's better. Will make the update.